### DISCUSSION TITLE
Lunch orders

### DISCUSSION THREAD

**Discussion by rebecca-vanhoeck** [SAEL-SWFSC/PAM-SI-Mtg-March2024#76] [Announcements]
🥪🥤 Please place your lunch orders online for all three days during this morning’s break!

Day 1: Mr. Greek Food Truck or order online at [Parakeet Cafe La Jolla](https://www.parakeetcafe.com/la-jolla-menu/)
🔑 Details: The Food Truck will be outside SWFSC. 
For Parakeet Cafe - order pickup for 1:00 pm on Wednesday 3/20 and reply to this announcement so we know how many orders there are.

Day 2: The Cheese Shop - [Menu](https://cheeseshoplajolla.com/la-jolla-la-jolla-shores-cheese-shop-food-menu)
🔑 Details: Add NOAA before your first Name
 “Schedule for Later” with Delivery at 12 pm on Thursday 3/21. 

Day 3: Marketplace Grille - [Menu](https://www.marketplacegrille.com/menu)
🔑 Details: Schedule for later with Delivery at 12pm on Friday 3/22

**Comment by Kourtney-Burger** [comment]
[Food Truck](https://www.instagram.com/mrgreeklunchtruck/)

**Comment by TaikiSan21** [comment]
Parakeet Cafe Order # 129 12:55pm
Cheese shop Order #3649171 12:00pm
Market Grille: Order #: 9166 3/22 @ 12:00pm

**Comment by DanWoodrichNOAA** [comment]
Parakeet order: 

chicken sandwich

Order Number

134

Scheduled Pickup

Wednesday, 3/20, 12:55 PM

Pickup Location

PARAKEET CAFE LA JOLLA

927 Silverado St., La Jolla, California, 92037

**Comment by chomweaverNOAA** [comment]
3/20-I have my own lunch
3/21 Cheese shop order- 12:00 pm, Order number 3649138
3/22-Wont be here

**Comment by carriecwall** [comment]
3/20: Parakeet Cafe Order #: 132 1pm; pick up for Carrie Bell
3/21: Cheese Shop Order #: 3649155 3/21 at 12pm; pick up for NOAA Carrie Bell
3/22: Market Place Grill web order #: 9152 3/22 at 12pm; pick up for Carrie Bell

**Comment by ybarkley** [comment]
Parakeet Cafe Order #141

**Comment by cberchok** [comment]
Ordered from Parakeet - I don't have an order number in the email.  Catherine Berchok for 1:05
Cheese shop #3649084 NOAA Catherine Berchok
Grille order # 9151- NOAA Catherine Berchok

**Comment by sahav** [comment]
Parakeet order #139 and #144 (three entrees + 2 drinks for Gen Davis)

**Comment by sahav** [comment]
Cheese shop order #3649041 (NOAA Samara Haver)
Marketplace grille: #9158 (NOAA Samara Haver)

**Comment by emmonsca** [comment]
Parakeet Cafe order 147
Cheese Shop #3649554
Marketplace grille #: 9157

**Comment by namariD** [comment]
Lunch 3/20= order #131, All I AVO Wanted (gf, veg)
Lunch 3/21= 1 Roast Beef Sandwich (Bread type:Sourdough, Add Cheese:Add Cheddar), 1 Chocolate Chip Cookie
Lunch 3/22= order #9154, Falafel
No Iceberg Lettuce
Basmati White Rice

**Comment by eoleson** [comment]
Marketplace grill order #: 9187- Crispy chicken bowl.

**Comment by MelissaSoldevilla** [comment]
Marketplace grill order: #: 9190 shrimp bowl
